lib/core/src/Ouroboros/Network/Client/Wallet.hs

```diff
@@ -81,7 +81,7 @@ import Network.TypedProtocol.Pipelined
     ( N (..), Nat (..), natToInt )
 import Numeric.Natural
     ( Natural )
-import Ouroboros.Consensus.Ledger.Abstract
+import Ouroboros.Consensus.HardFork.Combinator.Ledger.Query
```

Comment:
Seems like it changes location at every release, hide and seek 2.0 :upside_down_face:

Reply:
Yes. As far as I can tell, HFC is not yet covered by `Cardano.Api`.